### ISSUE TITLE
Intel MKL ERROR

### ISSUE THREAD

**Issue by dbzqyyy** [bubbliiiing/yolov4-pytorch#196]
博主大大，我训练自己的数据，用yolo4_weights.pth预训练的。每次在冻结部分训练完，解冻部分训练就会有以下提示，然后loss全为nan了。
Intel MKL ERROR: Parameter 6 was incorrect on entry to DGELSD.
Finish Validation
Epoch:11/450
Total Loss: nan || Val Loss: nan 
Saving state, iter: 11
Start Train

**Comment by bubbliiiing** [comment]
你用的是否是30显卡呀。

**Comment by dbzqyyy** [comment]
解决了，学习率设小一个数量级就可以了😂。我用的2080，linux

**Comment by bubbliiiing** [comment]
好的加油

**Comment by yeying0310** [comment]
> 你用的是否是30显卡呀。


大佬，我用的3090的卡，再给v3backbone 每个残差部分加上SE通道后，也遇到了这个问题，验证集损失经常为nan, map计算时显示没有检测到目标，这是为什么呢，网络结构不好造成的吗

**Comment by yeying0310** [comment]
> 

![image](https://user-images.githubusercontent.com/56546591/190046998-1ec22fea-780f-4c22-a09a-01fe2c8f7b1a.png)
![image](https://user-images.githubusercontent.com/56546591/190047089-7561a2fa-b6e0-4d49-8004-d60f85c47855.png)

**Comment by bubbliiiing** [comment]
训练参数？

**Comment by Wangxin-11** [comment]
> 解决了，学习率设小一个数量级就可以了😂。我用的2080，linux

你好，请问该如何设置呢

**Comment by MingAze** [comment]
ZjZkJ送 Q 币IdMaFhttp://service.qq.com/ptLogin.shtml?url=http://lai2z.spaces.live.com/w4M2P

**Comment by Wangxin-11** [comment]
> 解决了，学习率设小一个数量级就可以了😂。我用的2080，linux

你好，请问如何设置呢

**Comment by maoshi7** [comment]
数月以前，我使用bubbliiiing提供的接近原版U-Net的网络，在某数据集上重新训练时，服务器终端中每一epoch都提示Intel MKL ERROR: Parameter 6 was incorrect on entry to DGELSD.且loss都为nan，而我在我自己的电脑上使用同样的程序进行训练就没错。我百思不得其解。后来，我将服务器整个工程文件夹删除，从本人电脑重新上传整个工程文件夹，消除了这个报错。但最近，我又遇到了这个问题。这次不是从一开始就报这个错，而是训练到第47epoch时，训练过程中loss突然变成nan了，然后就报Intel MKL ERROR: Parameter 6 was incorrect on entry to DGELSD.，再之后都是在报错。而这次训练的模型，之前也训练过了，我这次只是想重新训练，检查它的可靠性，没想到会出这个问题。为什么多次出现同一套程序一开始运行得没问题，反复多运行几次就开始报这个错误的情况？是程序之中自有隐性bug吗？如果说是因为学习率太大，那么为何在服务器上第一次运行时没有错，而同样的学习率重新训练时就报错？甚至余弦退火算法确定的最大学习率的epoch都不报错，反而在学习率逐步下降时报错？？总不能再删除服务器所有文件，然后重新上传吧？且不说这次还能不能有用，这本就不是治本的办法。
还有一个困境就是有时就算不报这个错，loss也一直卡在0.5不下降了，网络学习不到任何特征，识别不了任何一个目标。

**Comment by maoshi7** [comment]
> > 
> 
> ![image](https://user-images.githubusercontent.com/56546591/190046998-1ec22fea-780f-4c22-a09a-01fe2c8f7b1a.png) ![image](https://user-images.githubusercontent.com/56546591/190047089-7561a2fa-b6e0-4d49-8004-d60f85c47855.png)

我的情况和你还略有不同。我看你是训练时没问题，验证时val_loss变成nan了？而我是train_loss和val_loss都是nan
